src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/Person.kt

```diff
@@ -137,7 +137,7 @@ data class Person(
         Contact.from(ContactType.MOBILE, prisoner.getMobilePhone()),
       )
       val contacts: List<Contact> = emails + phoneNumbers
-      val addresses: List<Address> = prisoner.addresses.map { Address(it.postcode) }
+      val addresses: List<Address> = prisoner.addresses.map { Address(it.postcode, it.fullAddress) }
```

Comment:
Could create a `Address.from(addresses: List<PrisonerAddressL)` helper function to do this